release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,145 @@ index a90ab14ff..f7f6f1a71 100644
+
+* Sometimes, during ZTP GitOps pipeline {sno} installations, the Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state. (link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])
+
+* When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang can occur, causing the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])
+
+* When booting ZT system machines from a live ISO with static IPv6 address configuration, `NetworkManager` exits successfully before the interface link becomes ready, which leaves the network interface without configuration. As a workaround, edit the RHCOS ISO referenced in the `AgentServiceConfig` CR to append `rd.net.timeout.carrier` to the kernel parameters in the `grub.cfg` file:
+
+. Pull the `rhcos-live` ISO image for the release being installed. The URL can be retrieved from the `AgentServiceConfig` CR on the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get AgentServiceConfig agent -o yaml
+----
+
+. Mount the image in the `/mnt/iso/` directory:
++
+[source,terminal]
+----
+$ mount rhcos-live.x86_64.iso /mnt/iso/
+----
+
+. Create the `iso-grub-cfg/` directory and change to the directory:
++
+[source,terminal]
+----
+$ mkdir iso-grub-cfg/; pushd iso-grub-cfg/
+----
+
+. Copy the contents of the `/mnt/iso/` directory to your working directory:
++
+[source,terminal]
+----
+$ rsync -avH /mnt/iso/* .
+----
+
+. Open the GRUB configuration file:
++
+[source,terminal]
+----
+$ vim EFI/redhat/grub.cfg
+----
+
+.. Append the `rd.net.timeout.carrier=20` string to the `linux` boot line.
+
+. Return to the initial working directory:
++
+[source,terminal]
+----
+$ popd
+----
+
+. Generate the ISO file from the `iso-grub-cfg` directory:
++
+[source,terminal]
+----
+$ mkisofs -JR -graft-points -o rhcos-carrier-timeout.iso iso-grub-cfg
+----
+
+. Push the updated ISO image to the server that is accessible by the hub cluster.
+
+. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
++
+[source,terminal]
+----
+$ oc edit AgentServiceConfig agent
+----
+
+. Update the `url` field to point to the URL of the updated ISO image.
+
++
+(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])
+
+* A kernel crash occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
```

Comment:
```suggestion
* A kernel failure occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
```

Reply:
See the ISG entry for 'crash': https://www.ibm.com/docs/en/ibm-style?topic=word-usage